Build/Tools/Yasm/LICENSE.txt

```diff
@@ -0,0 +1,74 @@
+Redistribution and use in source and binary forms, with or without
```

Comment:
I dont think we should ship two versions of yasm with cosmos. We currently have a version at [https://github.com/CosmosOS/Cosmos/tree/master/source/Cosmos.Build.Tasks/tools/yasm/win](https://github.com/CosmosOS/Cosmos/tree/master/source/Cosmos.Build.Tasks/tools/yasm/win) . Can we add a step to the build process to copy it from there to Build/Tools/Yasm ?